tech.json

```diff
@@ -864,6 +864,15 @@
               "requires": [ "canShinespark" ],
               "note": "Manually triggering reserve energy during a shinespark to extend the distance travelled.",
               "devNote": "This tech shouldn't need to be added to strat requirements."
+            },
+            {
+              "name": "canShinesparkDeepStuck",
+              "requires": [ "canShinespark" ],
```

Comment:
`canMidairShinespark` fits a little better.

Reply:
Good point, changed it to use that now